conjur-oss/values.yaml

```diff
@@ -7,7 +7,7 @@
 # rather than setting these in a custom values YAML file. This avoids the
 # risk of leaving around residual values files containing this sensitive
 # information.
-
+dataKeySecretRef: 
```

Comment:
The linter is failing here. I think this needs to default to an empty string instead of null.